### PR TITLE
Add CoC and short form of Contrib guidelines 

### LINKED ISSUE #29: Add basic information about how to contribute to this repo
It's currently not clear how the community can contribute to improving documentation in this repo. Spawning this issue to track how we're adapting or making references to:
- [ ] [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md) from the Gaia repo
  - [ ] Add who should be pinged for reviews
- [ ] [Code of Conduct](https://github.com/cosmos/gaia/blob/main/CODE_OF_CONDUCT.md)

### PR DESCRIPTION
**Pull request by dcwalk**
Closes #29 

First pass on that issue, I tried to avoid replicating content in the gaia contrib so referenced it directly, not sure if you have a preferred template or other pattern to follow @shahankhatch 

Also unclear if adding who to request review from is required, as perhaps better to set `CODEOWNERS` as you do in gaia: https://github.com/cosmos/gaia/blob/main/.github/CODEOWNERS

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+`contact@interchain.berlin`.
```
Comment (hxrts) [inline CODE_OF_CONDUCT.md:64]: @okwme is this a valid email?

**Comment by hxrts** [comment]
LGTM, let's just double check that email before merging. I know we're also phasing out interchain.berlin emails in favor of interchain.io, so may want that reflected in our CoC as well.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (okwme) [reply CODE_OF_CONDUCT.md:64]:
```suggestion
`community@interchain.io`.
```

**Comment by okwme** [comment]
email updated, i think @hyphacoop needs to accept suggested changes tho

**Comment by dcwalk** [comment]
Thanks! accepted!

**Comment by dcwalk** [comment]
I'll add a `CODEOWNERS` and flip for review

**Pull request merged**